demo/index.js

```diff
@@ -41,8 +42,25 @@ app.get('/:component?', (req, res) => {
         components: demoUtils.getComponentsWithExamples('src')
     };
 
+    // allow .only in example folder name
+    model.examples.some((example) => {
+        if (example.name.includes('.only')) {
+            model.examples = model.examples.filter(ex => ex.name.includes('.only'));
+            return true;
+        }
+    });
+
     req.model = model;
 
+    const md = new MobileDetect(req.headers['user-agent']);
+    const lassoFlags = ['skin-ds6'];
+    if (md.mobile() || md.tablet()) {
+        lassoFlags.push('touch');
+    } else {
+        lassoFlags.push('no-touch');
```

Comment:
Is there a need for `no-touch`? It seems to me you could simply do wha you need with testing for the presence of `touch`. Lasso flags are meant to be boolean in nature.

Or, in the inverse, you could just test for whether it's _not_ mobile or tablet: `!md.mobile() && !md.tablet()`, then add a `no-touch` flag, since we're defaulting to mobile. Your Lasso would then just check for whether the `no-touch` flag is present or not (just like our attribute booleans).

Reply:
Just to note, I'm currently defaulting to `no-touch` because of this: https://github.com/eBay/ebayui-core/pull/81/files/5d866c53d1e5c6339fefe54be94580fe7311e356#diff-9142d1141b49ae232a58033f834bf93cR17

If we want to keep this slightly future-proof, it could help to have both flags available, and have each component be able to use neither/either/both. Similar to how applications use both `isSmall` and `isLarge`. That said, you're right that at the moment, using only a `touch` flag would work.